core/assets/bundles/bundle_de.properties

```
@@ -389,7 +389,7 @@ error.mapnotfound = Kartendatei nicht gefunden!
 error.io = Netzwerk-Ein-/Ausgabe-Fehler.
 error.any = Unbekannter Netzwerkfehler.
 error.bloom = Bloom konnte nicht initialisiert werden.\nEs kann sein, dass dein Gerät es nicht unterstützt.
-zone.groundZero.name = Ground Zero
+zone.groundZero.name = Nullpunkt
```

Not a good name for a zone/level.
```suggestion
zone.groundZero.name = Einschlagsort
```

---

Ye, that's better.